modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -949,6 +956,92 @@ components:
             required:
                 - name
             type: object
+        FragmentContentField:
+            properties:
+                id:
+                    type: string
+                link:
```

Comment:
this has to be fragmentLink, if the property links to a schema, should be called the same.